guides/error_handling.md

```diff
@@ -0,0 +1,39 @@
+# Error Handling
+
+This page guides you through handling errors in Oban.
+
+The basics: jobs can fail in expected or unexpected ways. To mark a job as failed, you can return `{:error, reason}` from a worker's [`perform/1` callback](`c:Oban.Worker.perform/1`), as documented in the `t:Oban.Worker.result/0` type. A job can also fail because of unexpected raised errors or exits.
```

Comment:
Total nitpick, but can we keep these hard wrapped for consistency? Some doc contributions aren't wrapped, older ones are wrapped at 80, and newer entries are wrapped at 98 (which is preferred because it matches the moduledocs).

Reply:
Absolutely, gonna wrap everything at 98.